src/engine/strat_engine/filesystem.rs

```diff
@@ -153,6 +153,21 @@ pub fn create_fs(dev_path: &Path) -> EngineResult<()> {
     Ok(())
 }
 
+pub fn grow_fs(dev_path: &Path) -> EngineResult<()> {
```

Comment:
Matching the existing style of a lot of functions that shouldn't even exist doesn't seem like a virtue.